UWPExample/App.xaml

```diff
@@ -86,22 +87,15 @@
                     <SolidColorBrush x:Key="TextFillColorSecondaryBrush" Color="{StaticResource TextFillColorSecondary}" />
                     <SolidColorBrush x:Key="TextFillColorDisabledBrush" Color="{StaticResource TextFillColorDisabled}" />
                 </ResourceDictionary>
-            </ResourceDictionary.ThemeDictionaries>--><!--
+            </ResourceDictionary.ThemeDictionaries>-->
+    <!--
         </ResourceDictionary>
     </Application.Resources>-->
 
     <Application.Resources>
         <controls:XamlControlsResources>
             <controls:XamlControlsResources.MergedDictionaries>
-                <ResourceDictionary Source="ms-appx:///UWPPackage/GroupBox_Resource.xaml"/>
-                <ResourceDictionary Source="ms-appx:///UWPPackage/SettingsCard_Resource.xaml"/>
-                <ResourceDictionary Source="ms-appx:///UWPPackage/SettingsExpander_Resource.xaml"/>
-                <ResourceDictionary Source="ms-appx:///UWPPackage/Glyphs.xaml"/>
-                <ResourceDictionary Source="ms-appx:///UWPPackage/MarqueeText_Resource.xaml"/>
-                <ResourceDictionary Source="ms-appx:///UWPPackage/IconButton_Resource.xaml"/>
-                <ResourceDictionary Source="ms-appx:///UWPPackage/ProgressBarEx_Resource.xaml"/>
-                <ResourceDictionary Source="ms-appx:///UWPPackage/Segmented_Resource.xaml"/>
-                <!-- Other app resources here -->
```

Comment:
Removing this causes UWPExample (x64) fails to load the controls. 
![f040e1a717446727ed2f8a47cbf548e3](https://github.com/HO-COOH/WinUIEssentials/assets/42881734/9354ca0e-cf49-41a4-9ec3-b47af0b9b22c)